server/store/alert.go

```diff
@@ -0,0 +1,98 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	splunkAlertKey = "splunkalert"
+	splunkAlertMap = "splunkalertmap"
+)
+
+// AlertStore API for alert KVStore.
+type AlertStore interface {
+	GetAlertChannelID(alert string) (string, error)
+	GetChannelAlertIDs(channelID string) ([]string, error)
+	CreateAlert(channelID string, alertsID string) error
+	DeleteChannelAlert(channelID string, alertsID string) error
+}
+
+func keyWithChannelID(key, id string) string {
+	return fmt.Sprintf("%s_%s", key, id)
+}
```

Comment:
What is the value of passing `splunkAlertKey` to it every time?

Reply:
this function will generate a key with help of subscriptionKey and the channel id